src/SME.SGP.Aplicacao/Consultas/ConsultasNotasConceitos.cs

```diff
@@ -160,5 +160,16 @@ private async Task<string> ObterRfProfessorTitularDisciplina(string turmaCodigo,
 
             return string.Empty;
         }
+
+        public TipoNota ObterNotaTipo(DateTime dataAvaliacao, long turmaId)
+        {
+            var notaTipo = servicoDeNotasConceitos.TipoNotaPorAvaliacao(new AtividadeAvaliativa()
+            {
+                TurmaId = turmaId.ToString(),
+                DataAvaliacao = dataAvaliacao
+            });
+
+            return notaTipo.TipoNota;
```

Comment:
Se não localizar, vai dar erro. 
Verificar se o tipo de nota não é null, se for tem que tratar a mensagem, jogando um NegocioException